http-api/src/v1/stats.rs

```diff
@@ -0,0 +1,26 @@
+use axum::response::IntoResponse;
+use axum::routing::get;
+use axum::{Extension, Json, Router};
+use librad::git::identities::{self, SomeIdentity};
+use serde_json::json;
+
+use crate::Context;
+use crate::Error;
+
+pub fn router(ctx: Context) -> Router {
+    Router::new()
+        .route("/stats", get(stats_handler))
+        .layer(Extension(ctx))
+}
+
+/// Return the stats for the node.
+/// `GET /stats`
+async fn stats_handler(Extension(ctx): Extension<Context>) -> impl IntoResponse {
+    let storage = ctx.storage().await?;
+    let project_count = identities::any::list(storage.read_only())
+        .map_err(Error::from)?
+        .filter(|res| matches!(res, Ok(SomeIdentity::Project(_))))
+        .count();
+
+    Ok::<_, Error>(Json(json!({ "project_count": project_count })))
```

Comment:
Perhaps we can create some hierarchy here and have `{ project: { count: 123 } }`, this creates the space for other stats in the future.

Reply:
Yeah for sure, wanted to add also the following we talked about
> we can have { projects: { count: 123 }, users: { count: 432 } }

Reply:
Done in [528f84e](https://github.com/radicle-dev/radicle-client-services/pull/182/commits/528f84eb6588d75324acc19caf9abaa30ca6b160)